tests/func/test_data_cloud.py

```diff
@@ -112,6 +115,21 @@ def _should_test_ssh():
     return True
 
 
+def _should_test_ssh_mocked(client):
+    if os.getenv("DVC_TEST_SSH") == "true":
+        return True
+
+    try:
+        if os.name == "nt":
```

Comment:
why have two different commands to test on windows and *nix? Why not use one? Also, the point of this PR is to make ssh test everywhere and always. So it doesn't need this function since it always should return True.

Reply:
How do you run `ls` command in Windows? AFAIK, the command is not supported.

Reply:
But `dir` works on linux :) At least that is how it is usually.

Reply:
Cool, I understand your point now ;)